Documentation/ResultsStructure.md

```diff
@@ -0,0 +1,150 @@
+| vM | muscle fibre lengthening velocity | mesh x muscles | [m/s] |
+| vMtilde | muscle fibre lengthening velocity (normalised to vMmax) | mesh x muscles | [1/s] |
+| lT | tendon length | mesh x muscles | [m] |
+| vT | tendon lengthening velocity | mesh x muscles | [m/s] |
+
+
+
+<img src="./Hill-type%20muscle%20model.svg" width="400" height="auto">
+
+Hill-type muscle
+
+
+
+### R.torque_actuators
+| Field | Description | Dimension | Unit |
+|------ | ----------- | ---- | ---- |
+| a | actuator activation | mesh x actuators | [-] |
+| e | actuator excitation | mesh x actuators | [-] |
+| T | actuator torque | mesh x actuators | [-] |
+
+
+### R.ground_reaction
+| Field | Description | Dimension | Unit |
+|------ | ----------- | ---- | ---- |
+| threshold | threshold for the vertical ground reaction force to detect stance | 1 x 1 | [N] |
+| initial_contact_side | side where initial contact was detected before post-processing | 1 x 1 | [text] |
+| GRF_r | ground reaction forces right side | mesh x 3 | [N] |
+| GRF_l | ground reaction forces left side | mesh x 3 | [N] |
+| GRM_r | ground reaction moments right side | mesh x 3 | [Nm] |
+| GRM_l | ground reaction moments left side | mesh x 3 | [Nm] |
+| COP_r | centre of pressure right side (in ground frame) | mesh x 3 | [m] |
+| COP_l | centre of pressure left side (in ground frame) | mesh x 3 | [m] |
+| idx_stance_r | indices of meshpoints with ground contact on right side | <mesh x 1 | [-] |
+| idx_stance_l | indices of meshpoints with ground contact on left side | <mesh x 1 | [-] |
+
+### R.misc
+| Field | Description | Dimension | Unit |
+|------ | ----------- | ---- | ---- |
+| body_mass | body mass | 1 x 1 | [kg] |
+| body_weight | body weight | 1 x 1 | [N] |
+
+### R.kinetics
+| Field | Description | Dimension | Unit |
+|------ | ----------- | ---- | ---- |
+| T_ID | inverse dynamics (inertial and contact) | mesh x coordinates | [Nm or N] |
+| T_spring | coordinate moments by springs | mesh x coordinates | [Nm or N] |
+| T_damping | coordinate moments by dampers | mesh x coordinates | [Nm or N] |
+| T_limit | coordinate moments by limit torques | mesh x coordinates | [Nm or N] |
+
+### R.ligaments
+| Field | Description | Dimension | Unit |
+|------ | ----------- | ---- | ---- |
+| l | ligament length | mesh x ligaments | [m] |
+| v | ligament lengthening velocity | mesh x ligaments | [m/s] |
+| F | ligament force | mesh x ligaments | [N] |
+| strain | ligament strain | mesh x ligaments | [%] |
+| stress | ligament stress | mesh x ligaments | [MPa] |
+| moment | coordinate moment (or force) due to ligaments | mesh x coordinates | [Nm or N] |
+| moment | coordinate power due to ligaments | mesh x coordinates | [W] |
+
+### R.spatiotemp
+| Field | Description | Dimension | Unit |
+|------ | ----------- | ---- | ---- |
+| step_length_r | step length of right leg | 1 x 1 | [m] |
+| step_length_l | step length of left leg | 1 x 1 | [m] |
```

Comment:
I see these are empty in the Dhondt_2024_4seg reference results?

Reply:
Yes, these are related to the constraints on step length.

Reply:
the name `step_length_r` would make me think this variable gives the step length, not it being dependent on constraints.

Reply:
It uses the same casadi function. I'm removing them from the overview, and getting rid of the empty field if it's not calculated.